### PR TITLE
fixes#5 : Adds scholar, counter component

### LINKED ISSUE #5: Create Crousal 2

### PR DESCRIPTION
**Pull request by sanjana-302**
- [x] Closes #5
---

## PR description

- Adds Scholar component
-  Adds Counter component
- Update component name (Datetable) to CamelCase.
- Screenshot from 2021-08-13 15-43-48
- Screenshot from 2021-08-13 15-43-56
- Screenshot from 2021-08-13 15-48-01
- Screenshot from 2021-08-13 15-48-09

### CODE REVIEW

File: components/Counter.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    return (
+      <div className="flex mx-2 my-5 lg:my-40 md:my-14 lg:mx-20 justify-around lg:text-4xl md:text-2xl text-blue-850 font-bold">
+        <div className="flex-shrink "> &nbsp;&nbsp;&nbsp;&nbsp; {"22"} <br/> {"Countries"}</div>
+        <div className="flex-shrink">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{"100"} <br/> {"Universities"}</div>
+        <div className="flex-shrink">&nbsp;&nbsp;&nbsp;&nbsp;{"120"} <br/> {"Scholars"}</div>
+      </div>
+    );
+  };
+  
+  export default Counter;
```
Comment (pnijhara) [inline components/Counter.js:11]: Add an empty line at the end of the file.

File: components/Scholar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div>
+      <div className="  text-blue-850 font-bold lg:text-3xl mt-2">
+        Our Scholars
+      </div>
+      <div className="container container max-w-full m-auto flex flex-wrap flex-col md:flex-row items-center justify-start">
+        <div className="w-full pt-4">
+          <div className="flex flex-col lg:flex-row overflow-hidden border-2 border-gray-300">
+            <Image
+              width={1650}
```
Comment (pnijhara) [inline components/Scholar.js:13]: Decrease the image size and increase the content size.

File: components/Scholar.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+              height={1350}
+              src="/scholar_one.jpg"
+              alt="not found"
```
Comment (pnijhara) [inline components/Scholar.js:16]: Always write alt for the images.

**Review by pnijhara — CHANGES_REQUESTED** [review]
Please look at the requested changes. Let me know if you can implement a carousel in them. If not, we can merge this. We can open another issue for a carousel.

LGTM otherwise.

**Comment by sanjana-302** [comment]
For the carousel, I think we can open a separate issue. It may take some time as what I implemented is still not functioning properly. I will add all other requested changes.

**Review by pnijhara — APPROVED** [review]

**Pull request merged**